package/MDAnalysis/analysis/dihedrals.py

```diff
@@ -372,8 +420,10 @@ def _conclude(self):
         self.angles = np.rad2deg(np.array(self.angles))
 
     def plot(self, ax=None, ref=False, **kwargs):
-        """Plots data into standard ramachandran plot. Each time step in
-        :attr:`Ramachandran.angles` is plotted onto the same graph.
+        """Plots data into standard ramachandran plot.
```

Comment:
```suggestion
        """Plots data into standard Ramachandran plot.
```

(this was my mistake!)